docs/guides/getting-started-java-spring.md

```diff
@@ -0,0 +1,256 @@
+---
+id: getting-started-java-spring
+title: Getting started as a Java developer using Spring
+sidebar_label: Getting started as a Java developer using Spring
+description: "Use Spring Boot and the Spring Zeebe SDK to interact with your local Self-Managed Camunda 8 installation."
+keywords: [java, spring, spring zeebe, getting started, user guide, tutorial]
+---
+
+<span class="badge badge--beginner">Beginner</span>
+<span class="badge badge--medium">1 hour</span>
+
+In this guide, we'll step through using Spring Boot and the [Spring Zeebe SDK](/apis-tools/spring-zeebe-sdk/getting-started.md) to interact with your local Self-Managed Camunda 8 installation.
+
+By the end of this tutorial, you'll be able to use Spring and Java code with Zeebe to:
+
+- Deploy a process model.
+- Initiate a process instance.
+- Handle a service task.
+
+For example, in this guide we will outline a BPMN model to receive a payment request, prepare a transaction, charge a credit card, and execute a payment:
+
+![example BPMN model to receive a payment request, prepare a transaction, charge a credit card, and execute a payment](./img/prepare-transaction-example.png)
+
+:::note
+This tutorial is not intended for production purposes.
+:::
+
+## Prerequisites
+
+Before getting started, ensure you:
+
+- Can access to your preferred code editor or IDE.
+- Have Java [installed locally](https://www.java.com/en/download/).
```

Comment:
I'm debating whether to recommend the Java version here or link to the Supported Env page. This would guide them to the OpenJDK version information, if they are interested.

Reply:
Resolved in latest commit 👍